sormas-api/src/main/java/de/symeda/sormas/api/utils/DateHelper.java

```diff
@@ -186,15 +192,42 @@ public static Date parseDateTimeWithException(String date, String dateTimeFormat
 	}
 
 	private static Date parseDateWithException(String date, List<String> dateFormats) throws ParseException {
+
 		if (date == null) {
 			return null;
 		}
 
+		Logger logger = LoggerFactory.getLogger(DateHelper.class);
 		for (String format : dateFormats) {
 			try {
-				DateTimeFormatter formatter = DateTimeFormat.forPattern(format);
-				return DateTime.parse(date, formatter).toDate();
-			} catch (Exception e) {
+				DateTimeFormatter formatter = new DateTimeFormatterBuilder().parseCaseInsensitive()
```

Comment:
I think we have to handle this differently.
The method is typically called with a format provided by the user language, e.g. I18nProperties.getUserLanguage().getDateTimeFormat()`, so normally this should work, but it could also be called with a format provided by another language and in this case the language of the provided date format would not match the language used here.
I feel that I don't fully understand what the locale provided to the formatter actually does. Shouldn't the date format already contain the language specific stuff like "-" vs. "."?

Reply:
As discussed with @MartinWahnschaffe the parse methods now take `Language` as parameter to combine format String and Locale.